src/components/view/ContactIntake/index.js

```diff
@@ -1,3 +1,497 @@
-import IntakeForm from './IntakeForm';
+import React, { Component } from 'react'
 
-export default IntakeForm;
\ No newline at end of file
+import muiThemeable from 'material-ui/styles/muiThemeable';
+
+import TextField from 'material-ui/TextField'
+import FlatButton from 'material-ui/FlatButton'
+import Checkbox from 'material-ui/Checkbox';
+import Toggle from 'material-ui/Toggle';
+import Slider from 'material-ui/Slider';
+import { Card, CardHeader, CardTitle, CardText } from 'material-ui/Card';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+import { RadioButton, RadioButtonGroup} from 'material-ui/RadioButton';
+import DatePicker from 'material-ui/DatePicker';
+
+// TODO: this should be mapped via props instead
+import { createEvent } from 'actions';
+
+import PeriodicQuestionsForm from 'components/view/ContactIntake/periodicQuestions.form';
+import NewContactQuestionsForm from 'components/view/ContactIntake/newContactQuestions.form';
+import OutreachQuestionsForm from 'components/view/ContactIntake/outreachQuestions.form';
+import StandardQuestionsForm from 'components/view/ContactIntake/standardQuestions.form';
+
+import * as moment from 'moment';
+
+import './styles.css';
+
+
+class IntakeForm extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.submitForm = this.submitForm.bind(this)
+
+        const todayDate = new Date();
+        todayDate.setFullYear(todayDate.getFullYear());
+        todayDate.setHours(0, 0, 0, 0);
+
+        const initDateOfBirth = new Date(1980, 0, 1)
+
+        // grab location if we can
+        if (navigator.geolocation) {
+            navigator.geolocation.getCurrentPosition((position) => {
+                let stateLocationObject = {...this.state.eventLocation};
+                stateLocationObject.latitude = position.coords.latitude;
+                stateLocationObject.longitude = position.coords.longitude;
+                this.setState({eventLocation: stateLocationObject});
+            });
+        };
+
+        const { match: { params } } = this.props;
+
+        // TODO: prepopulate with personal info
+        this.state = {
+
+            showPeriodic: false,
+            showOutreach: false,
+            showNewContactQuestions: false,
+
+            eventNotes: '',
+
+            // form
+            eventDate: todayDate,
+
+
+            // periodic
+            housingStatus: 'homeless',
+            hivStatus: 'neverTested',
+            isInCareForHiv: false,
+            hepCStatus: 'neverTested',
+            isInCareForHepC: false,
+            healthInsurer: 'MH',
```

Comment:
Are we posting Mass Health as a default, even if health insurance is not toggled?

Reply:
good point. fixed